index.bs

```diff
@@ -5224,67 +5380,68 @@ The [=remote end event trigger=] is:
 
 <div algorithm="remote end event trigger for log.entryAdded">
 
-Define the following [=console steps=] with |method|, |args|, and <var
-ignore>options</var>:
+Define the following [=console steps=] with |method|, |args|, and
```

Comment:
There is a lot of refactoring here for the console steps which would have been great to avoid and only include the calling convention for the object serialization. But now it's already here... so I'm ok.

Reply:
I think GitHub's diff is just terrible and what's changed is that we need to know the session to do the serialization, so we move that loop to the outside.

Reply:
Maybe. It was also more a comment so I'm going to close the issue.